packages/data-models/flowTypes.ts

```diff
@@ -377,7 +377,6 @@ export namespace FlowTypes {
     | "parent_point_box"
```

Comment:
remove previous `group` template type that was used to display items (no longer required as items do not render in groups, and also the component is too easily be confused with display_groups)